packages/mdc-ripple/index.js

```diff
@@ -114,3 +114,12 @@ export class MDCRipple extends MDCComponent {
     this.unbounded = 'mdcRippleIsUnbounded' in this.root_.dataset;
   }
 }
+
+/**
+ * @typedef {!{
+ *   root_: !Element,
+ *   unbounded: (boolean|undefined),
+ *   disabled: (boolean|undefined)
+ * }}
+ */
+let RippleAbleType;
```

Comment:
I talked to our MD team a bit, and I think "RippleCapableSurface" is more clear. Can you add a link to our guidelines on when to use ripples:
https://material.io/guidelines/motion/choreography.html#choreography-radial-reaction

Can you also add a note about what "unbounded" and "disabled" means from a ripple's point of view.